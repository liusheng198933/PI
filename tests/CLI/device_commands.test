#!/bin/sh
./run_one_test.py $srcdir/testdata device_commands simple_router.json
