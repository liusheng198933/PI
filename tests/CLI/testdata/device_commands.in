show_devices
remove_device 0
show_devices
