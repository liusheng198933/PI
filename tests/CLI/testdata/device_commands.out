????
Device assigned successfully.
Selecting device.
????
Showing devices:
0 (selected)
????
Device removed successfully.
????
Showing devices:
????
